distribution/docs/src/main/resources/content/_metadataReference/core-attributes-table.adoc

```diff
@@ -78,13 +78,13 @@ __Coordinates must be in *lon-lat* coordinate order__
 |
 
 |[[_resource-uri]]resource-uri
-|Location of the resource for the metacard.
+|Catalog-specific location of the resource for the metacard. This URI is used for internal catalog requests.
 |String
 |Valid URI per RFC 2396
 |
 
 |[[_resource-download-url]]resource-download-url
-|URL location of the resource for the metacard. This attributes provides a resolvable URL to the download location of the resource.
+|URL location of the resource for the metacard. This attributes provides a client-resolvable URL to the download location of the resource. Clients should use this URL for download requests.
```

Comment:
@pklinef  @lessarderic  @peterhuffer What do you think about providing guidance in another part of the documentation about how, why or when the `download-url` is populated? I have not looked in the code, so couldn't say. Did the download-url attribute get added because of reverse proxies?

Reply:
It is handled by a plugin so it should always be present if a `resource-uri` is present.  Ideally they should just request to download using the REST endpoint and the ID.  That was the only supported method originally.